src/pyrqlite/connections.py

```diff
@@ -95,7 +95,7 @@ def connect_timeout(self):
 
     def _init_connection(self):
         timeout = None if self.timeout is None else float(self.timeout)
-        if self.scheme in ('http', ':memory:'):
+        if self.scheme == "http":
```

Comment:
':memory:' is used by the unit tests

Reply:
Ah, OK, the unit test suite is a standard thing. Got it. I was wondering why (hadn't looked into it yet).